src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Route, Router } from 'react-router';
 import ReactGA from 'react-ga';
+import ScrollToTop from 'ScrollToTop';
```

Comment:
Will you please move this to `src/shared/components/` and fix the import to reflect that movement?